Makefile

```diff
@@ -27,9 +27,11 @@ proto:
 	done
 
 test: testdeps
+	go test -i google.golang.org/grpc/...
```

Comment:
It does not fix ... Even if it fixes, it requires a lot of users doing that, which is not good. Perhaps it is not worth doing that.